storage/src/adb/any/variable/mod.rs

```diff
@@ -725,11 +725,10 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T: Translato
             );
         }
 
-        let section = self.current_section();
         try_join!(
-            self.mmr.sync(&mut self.hasher).map_err(Error::Mmr),
-            self.log.sync(section).map_err(Error::Journal),
-            self.locations.sync().map_err(Error::Journal),
```

Comment:
Not directly related to this PR but these should have been calls to close, even if it is all the same in the end.